_unit-test/ensure-relay-credentials-test.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+source "$(dirname $0)/_test_setup.sh"
+source dc-detect-version.sh
```

Comment:
git compare here isn't working properly, but this line was added and is needed since this test was failing due to changes months ago with the addition of `$dc` and `$dcr`. 

https://github.com/getsentry/self-hosted/issues/1289
https://github.com/getsentry/self-hosted/issues/1301